### DISCUSSION TITLE
Microsoft Form Check

### DISCUSSION THREAD

**Discussion by XxMacexX** [silverhack/monkey365#63] [Q&A]
Hi,
On your screenshot you can see Microsoft Forms, how is this queried?
best regards

**Comment by silverhack** [comment]
Hey @XxMacexX, 

There is only one check for Microsoft Forms and is executed when the Microsoft365 option is selected. You can find available options [here](https://silverhack.github.io/monkey365/configuration/microsoft365-options).

Cheers,

**Reply by XxMacexX** [reply]
I've seen it bypass Purview and Microsoft365 for me. I can only check Teams and Exchange. My command is the following: 
$assets = Invoke-Monkey365 -ExportTo HTML -PromptBehavior SelectAccount -IncludeAzureAD -Instance Microsoft365 -Analysis ExchangeOnline,Microsoft365,PurView,MicrosoftTeams,SharePointOnline

I don't get an error message. 
Just a warning when I check Purview: WARNING: Your connection has been redirected to the following URI:
"https://eur01b.ps.compliance.protection.outlook.com/Powershell-LiveId?BasicAuthToOAuthConversion=true;PSVersion=5.1.22
000.1335"
